ucfcs/GrowthPlus
# Julio/delete child

This PR implements delete child from realm functionality from within the child screen. 
To test: 
Add some new children, test the adding and navigation between screen.
Navigate to the parent portal and click on a child card and delete it. 
The app should go back to the parent portal where the deleted child will no longer be there. 
Go back to the landing page and inspect the deleted child is no longer there either. 

Note: 
You can try deleting the pre-defined data that we get from the json file but it will re-populate again so don't be alarm if it comes back. It is better to test it with newly created children, the sample realm data will be taken off after I update the schemas and we are able to map the parent to the children.
